src/components/transactions/delegateStatistics.js

```diff
@@ -11,7 +11,7 @@ class DelegateStatistics extends React.Component {
 
     let status = '';
     if (delegate && delegate.rank) {
-      status = delegate.rank < 101 ? this.props.t('Active') : this.props.t('Standby');
+      status = delegate.rank <= 101 ? this.props.t('Active') : this.props.t('Standby');
```

Comment:
not belonging to the change itself but will be ideal to use constants/voting.maxCountOfVotes